components/TextFields/tests/snapshot/supplemental/MDCAbstractTextFieldSnapshotTests+I18N.m

```diff
@@ -20,6 +20,34 @@
 
 @implementation MDCAbstractTextFieldSnapshotTests (I18N)
 
+- (void)MDCForceTextInputRightToLeft:(UIView *)view {
+  if ([view conformsToProtocol:@protocol(UITextInput)]) {
+    id<UITextInput> textInput = (id<UITextInput>)view;
+    UITextRange *textRange = [textInput textRangeFromPosition:textInput.beginningOfDocument
+                                                   toPosition:textInput.endOfDocument];
+    if (textRange) {
+      [textInput setBaseWritingDirection:UITextWritingDirectionRightToLeft forRange:textRange];
+    } else {
+      XCTAssertNotNil(textRange);
```

Comment:
I might misunderstand things here. What are `textRange` expected to look like here?

Reply:
Followed up offline, we want an assertion failure to be thrown if the `else` block is executed. And because XCTAssert doesn't stop execution. Moving this line above the `setBase..` would still require the `if (textRange)` guard. Putting a comment there would be clear:)

Reply:
Done